libs/components/phone-field/testing/src/phonefield/phone-field-fixture.spec.ts

```diff
@@ -165,6 +165,8 @@ describe('PhoneField fixture', () => {
   });
 
   it('should use newly selected country', async () => {
+    fixture.detectChanges();
+    await fixture.whenStable();
```

Comment:
I removed these changes locally and ran `npx nx test phone-field-testing` and all was well. Do we still need them?

Reply:
Removed - those were from an earlier version.